app/src/components/ErrorBoundary.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { ReactNode } from "react";
+import { css } from "@emotion/react";
+
+import { EmptyGraphic, Flex, View } from "@arizeai/components";
+
+import { ExternalLink } from "./ExternalLink";
 type ErrorBoundaryProps = { children: ReactNode };
 export class ErrorBoundary extends React.Component<
   ErrorBoundaryProps,
-  { hasError: boolean }
+  { hasError: boolean; error: unknown }
```

Comment:
whoa why the class here? just copy pasta?

Reply:
Error boundaries are classes https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary

Reply:
ahh gotcha cause you have to implement the getDerivedStateFromError, makes sense!